resources/views/home.blade.php

```diff
@@ -620,11 +620,11 @@ class="h-[8rem]"
                     </x-slot>
                 </x-testimonial>
 
-                <x-testimonial url="https://twitter.com/zepfietje/status/1486763133591072768">
-                    Need an admin panel for your Laravel project? It's <strong>powerful</strong>. It's <strong>fast</strong>. It's <strong>beautiful</strong>. It's <strong>easy</strong>.
+                <x-testimonial url="https://twitter.com/nkornel/status/1547530825117360129">
+                    We chose Filament and <strong>clients love it</strong>. It is <strong>consistent</strong> and <strong>clean</strong> so the <strong>learning curve is better</strong>.
 
-                    <x-slot name="author" avatar="https://avatars.githubusercontent.com/u/44533235?v=4">
```

Comment:
Lol yeah was also thinking it's probably good to remove my testimonial now I'm part of the team, @danharrin. 😂